dart/test/http_client/breadcrumb_client_test.dart

```diff
@@ -202,8 +202,8 @@ void main() {
 class CloseableMockClient extends Mock implements BaseClient {}
 
 class Fixture {
-  BreadcrumbClient getSut([MockClient? client]) {
-    final mc = client ?? getClient();
+  BreadcrumbClient getSut(MockClient client) {
+    final mc = client;
```

Comment:
No need of defining another var `mc` in this case, you can use `client` directly, works for all the others too.